jscomp/others/belt.res

```diff
@@ -1,4 +1,4 @@
-(* Copyright (C) 2015-2016 Bloomberg Finance L.P.
+@@ocaml.text(/* Copyright (C) 2015-2016 Bloomberg Finance L.P.
```

Comment:
Why does this not reprint to our /*** */ syntax? same for the function based docstrings?

Reply:
That's the part I had meant to leave to you 😄 as I saw that you had done something in that direction instead https://gist.github.com/ryyppy/a0094cbda52ff57b86fdb0bdcba1edb6.

It seems that gist is not up to date and it needs to be `@@res.doc` / `@res.doc` instead. But that's not the only issue, it's also that all double quotes in the comments were escaped during the conversion.

I'll look into it.

Reply:
this was a while ago and didn't remember all the details on how the printing worked. my current conversion PR is having conflicts with main and i am having trouble figuring out the relevant changes. so might as well start from scratch with my script :-/